18_regexp_lab/02.py

```diff
@@ -1,6 +1,10 @@
-"""
-Write a function toCamelCase(text) that returns the text camel cased
-Write a function to_underscore(text) that returns the text with 
-underscores between words
-"""
+#!/usr/bin/python
+import re 
+import sys, os
+text = 'no_more_shall_we_part'
+def under_scope_to_camelCase(text):
+	x = re.sub(r'\_[a-z]' ,lambda m: m.group(0).upper()  , text)
+	x = re.sub(r'_' , '' , x)
```

Comment:
A simpler one- `re.sub(r'_([a-z])', lambda m: m.group(1).upper(), text)`

And then you don't need to remove the leftover underscore